### PR TITLE
Update Traffic Limits doc

### LINKED ISSUE #5596: how to block IPs
**Note:** Partial or incorrectly filled out issues may be deferred.

Re: Add to #5545 

Priority: Low/Medium/High (choose one, remove the other options)

## Issue Description
customers could use information about how to block IP ranges

## Suggested Resolution
document php or plugins to resolve

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes # DOC-62
Closes #5596

## Effect
PR includes the following changes:
- clarifies how Pantheon measures traffic
- better highlights that Dashboard metrics won't match Google Analytics and why
- provides resources that customers can use to investigate potential causes of unexpected traffic (new doc!)

## Remaining Work
- [x] integrate investigating resources that customers can use
- [x] add steps suggested by CS about blocking IP range
- [x] add new doc to cat page (don't forget the post-launch stuff)
- [x] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `59` | `47` | `-12` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18769/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18769/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/optimize-site-traffic.md [hunk 1]
```diff
@@ -22,16 +22,25 @@ Consult our doc for a list of [WordPress best practices](/wordpress-best-practic
...
+This issue affects both WordPress and Drupal sites, but the request path will vary between the two platforms. On WordPress, it often appears as a `favicon.ico` file in the root directory. In Drupal (specifically Drupal 8), it shows up as a system path.
 
-Often, Distributed Denial of Service (DDoS) attacks are short-lived and unlikely to be a prolonged issue. Our [Customer Success](https://pantheon.io/docs/support) team is available to assist with identifying a DDoS attempt, and take steps to mitigate it for your site.
+|  **CMS**  |          **Path**         |
+|:---------:|:-------------------------:|
+| WordPress | /favicon.ico              |
+| Drupal    | /system/files/favicon.ico |
 
-## Favicon.ico Generates Page Visits
+**Solution**: Often the fix is to add and commit a static `favicon.ico` into the path that is being requested. What is usually the actual culprit is adding a custom favicon through the active theme for the site through some kind of upload button, and then the icon is deleted or some other kind of issue which causes the CMS to look for an alternative favicon.
```
Comment (kyletaylored) [inline source/content/optimize-site-traffic.md]: Would replace **button** with **form** (my mistake on the writeup)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `43` | `-13` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18818/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18818/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `61` | `40` | `-21` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18854/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18854/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `61` | `50` | `-11` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18858/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18858/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `49` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18862/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18862/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `51` | `-9` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19263/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19263/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `59` | `3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19318/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19318/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `54` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19334/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19334/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `57.99999999999999` | `4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19336/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19336/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `48` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19678/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19678/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `42` | `44` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19804/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19804/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5545 | Difference |
| --- | --- | --- | --- |
| Performance Score | `53` | `45` | `-8` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5545` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19813/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19813/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**